src/main/java/nextstep/qna/domain/QuestionBody.java

```diff
@@ -0,0 +1,10 @@
+package nextstep.qna.domain;
+
+public class QuestionBody {
+
+    private String value;
```

Comment:
```suggestion
    private String title;
    
    private String cotents;
```
이 두 개의 값을 가지는 객체를 의미했음